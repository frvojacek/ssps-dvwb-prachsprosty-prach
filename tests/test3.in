5 2
09000
32010
2
1 4 0 3
15 0 1 3