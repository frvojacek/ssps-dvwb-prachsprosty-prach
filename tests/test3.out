08000
21000

07123
10122

06012
00011

05001
00000

04000
00000

03000
00000

02000
00000

01000
00000

00000
00000

00000
00000

00000
00000

00000
00000

00000
00000

00000
00000

00000
00000

22100
32100

11000
21000

00000
10000

00000
00000
